be/src/vec/sink/writer/vfile_result_writer.cpp

```diff
@@ -172,9 +172,11 @@ Status VFileResultWriter::_create_file_writer(const std::string& file_name) {
 
 // file name format as: my_prefix_{fragment_instance_id}_0.csv
 Status VFileResultWriter::_get_next_file_name(std::string* file_name) {
+    std::string suffix =
+            _file_opts->file_suffix.empty() ? _file_format_to_name() : _file_opts->file_suffix;
     std::stringstream ss;
     ss << _file_opts->file_path << print_id(_fragment_instance_id) << "_" << (_file_idx++) << "."
```

Comment:
```suggestion
    ss << _file_opts->file_path << print_id(_fragment_instance_id) << "_" << (_file_idx++) << "." << _file_opts->file_suffix;
```
May be we can do this transformation in FE.
So, here can just write like this.

Reply:
If modified like this, there should be problems when only BE is upgraded?